daemon/create.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // ContainerCreate takes configs and creates a container.
-func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool) (types.ContainerCreateResponse, error) {
+func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool, fromBuilder bool) (types.ContainerCreateResponse, error) {
```

Comment:
Not sure I understand - this is called from both the server API and the builder. Are you asking for the function name to be renamed? If so, I don't mind what it's called - it's as it's been for ever. I just needed the context from which it's called passed through.